src/components/Story.svelte

```diff
@@ -1,60 +1,37 @@
 <script lang="ts">
-  import {LatLng, Map} from 'leaflet';
-  import { onDestroy } from 'svelte';
-  import Slider from '@smui/slider';
-  import { mapStore, layersStore, incomeSlider } from '../stores.js';
-  import Paper, { Title, Subtitle, Content } from '@smui/paper';
+    import {LatLng, Map} from 'leaflet';
+    import { onDestroy } from 'svelte';
+    import Accordion, { Panel, Header, Content } from '@smui-extra/accordion';
 
-  let map: Map|undefined = undefined;
+    import { mapStore } from '../stores';
+    import config, { MapState } from '../config';
 
-  const unsubscribe = mapStore.subscribe((newMap: Map|null) => {
-    if (newMap != null) {
-      map = newMap;
+
+    let map: Map|undefined = undefined;
+
+    const unsubscribe = mapStore.subscribe((newMap: Map|null) => {
+        if (newMap != null) {
+            map = newMap;
+        }
+    });
+
+    onDestroy(unsubscribe);
+
+    function setMapState(mapState: MapState): void {
+        map?.flyTo(new LatLng(mapState.lat, mapState.lng), mapState.zoom);
     }
-  });
 
-  onDestroy(unsubscribe);
 
-  function panTo(): void {
-    map?.flyTo(new LatLng(40.68967735303955, -74.04534588323135), 15);
-  }
 </script>
 
-<style>
-    .panel {
-        margin-bottom: 1rem;
-    }
-</style>
-
-<div class="panel">
-{#if $layersStore['Statue of Liberty']}
-<Paper>
-    <Title>Statue of Liberty</Title>
-    <Subtitle>Pan to the Statue of Libery</Subtitle>
-    <Content>
-        <button on:click={panTo}>Statue of Liberty</button>
-    </Content>
-</Paper>
-{/if}
-</div>
 
-<!-- hacky solution until I learn about the svelte css better. Otherwise, the if ignores the class -->
-<div class="panel">
-{#if $layersStore['New York City Income']}
-<Paper>
-    <Title>NYC INCOME - {$incomeSlider > 225 ? 'All' : `$${$incomeSlider * 1000}`}</Title>
-    <Subtitle>Select income. Darker areas mean higher percentage below slider</Subtitle>
-    <Content>
-        <Slider
-            bind:value={$incomeSlider}
-            min={0}
-            max={250}
-            step={5}
-            discrete
-            input$aria-label=""
-            color="secondary"
-        />
-    </Content>
-</Paper>
-{/if}
+<div class="accordion-container">
+    <Accordion>
+        {#each config.stories as story}
+            <Panel on:click={() => setMapState(story.mapState)}>

```

Comment:
Imagine I am searching around on the map. I am looking at NYC, then I see this amazing story by njason about the heat in the heights. I click it, and I am zoomed into to somewhere NJ. I didn't want that. Now I am lost on the map.

Doesn't have to happen here, but maybe we should think about some sort of "back" button, or alternatively putting some map state ideas into the url, similar to google maps.